### PR TITLE
[MRG] Linear Neural Networks/The Image Classification Dataset

### PR DESCRIPTION
**Pull request by xiaotinghe**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-556/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-556/

**Comment by mli** [comment]
Job d2l-zh/PR-556/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-556/

**Comment by mli** [comment]
Job d2l-zh/PR-556/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-556/

File: chapter_linear-networks/image-classification-dataset.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-# 影像分类数据集
+# 图像分类数据集
 :label:`sec_fashion_mnist`
 
-广泛使用的图像分类数据集之一是 MNIST 数据集 :cite:`LeCun.Bottou.Bengio.ea.1998`。虽然它作为基准数据集运行良好，但即使是按照当今标准的简单模型也能达到 95% 以上的分类准确率，因此不适合区分较强的模型和较弱的模型。如今，MNIST 的作用更多的是理智检查，而不是作为基准。到了赌注只是一点点, 我们将集中讨论在未来部分的质量相似, 但相对复杂的时尚多国主义数据集 :cite:`Xiao.Rasul.Vollgraf.2017`, 这是在 2017 年发布.
+目前广泛使用的图像分类数据集之一是 MNIST 数据集 :cite:`LeCun.Bottou.Bengio.ea.1998`。虽然它作为基准数据集运行良好，但按今天的标准，即使是简单的模型也能达到95%以上的分类准确率，因此不适合区分强模型和弱模型。如今，MNIST更像是一个健全检查，而不是一个基准。
```
Comment (astonzhang) [inline chapter_linear-networks/image-classification-dataset.md:4]: “运行良好”不妥帖

File: chapter_linear-networks/image-classification-dataset.md [hunk 2]
```diff
@@ -127,9 +128,10 @@ y = tf.constant(mnist_train[1][:18])
 show_images(X, 2, 9, titles=get_fashion_mnist_labels(y));
 ```
 
-## 读取小批处理
+## 读取小批量
 
-为了让我们在阅读训练和测试集时更轻松，我们使用内置的数据迭代器，而不是从头开始创建数据迭代器。回想一下，在每次迭代中，数据加载器每次读取大小为 `batch_size` 的小批数据。我们还随机洗牌训练数据迭代器的示例。
+为了使我们在读取训练集和测试集时更容易，我们使用内置的数据迭代器，而不是从零开始创建一个。
+回顾一下，在每次迭代中，数据加载器每次都会读取一小批量数据，大小为`batch_size`。我们在训练数据迭代器中还对所有样本进行了随机打乱。
```
Comment (astonzhang) [inline chapter_linear-networks/image-classification-dataset.md:134]: 我们在训练数据迭代器中还随机打乱了所有样本。

File: chapter_linear-networks/image-classification-dataset.md [hunk 3]
```diff
@@ -138,8 +140,8 @@ def get_dataloader_workers():  #@save
     """Use 4 processes to read the data expect for Windows."""
     return 0 if sys.platform.startswith('win') else 4
 
-# `ToTensor` converts the image data from uint8 to 32-bit floating point. It
-# divides all numbers by 255 so that all pixel values are between 0 and 1
+# 通过ToTensor实例将图像数据从uint8格式变换成32位浮点数格式
```
Comment (astonzhang) [inline chapter_linear-networks/image-classification-dataset.md:143]: 贪婪换行：当抵到78个字符时再换行

File: chapter_linear-networks/image-classification-dataset.md [hunk 4]
```diff
@@ -241,19 +242,19 @@ for X, y in train_iter:
     break
 ```
 
-我们现在已经准备好与时尚 MNist 数据集在下面的部分。
+我们现在准备在下面的章节中使用Fashion-MNIST数据集。
```
Comment (astonzhang) [inline chapter_linear-networks/image-classification-dataset.md:245]: 已经准备好

File: chapter_linear-networks/image-classification-dataset.md [hunk 5]
```diff
@@ -241,19 +242,19 @@ for X, y in train_iter:
...
 
-## 摘要
+## 总结
 
-* 时尚 MNist 是一个服装分类数据集，由代表 10 个类别的图像组成。我们将在后续章节和章节中使用此数据集来评估各种分类算法。
-* 我们将任何高度为 $h$ 像素的图像形状存储为 $w$ 像素。
-* 数据迭代器是高效性能的关键组件。依靠实施良好的数据迭代器，利用高性能计算来避免减慢训练循环。
+* Fashion-MNIST是一个服装分类数据集，由10 个类别的图像组成。我们将在后续章节中使用此数据集来评估各种分类算法。
+* 我们将高度$h$像素，宽度$w$像素图像的形状记为$h \times w$或（$h$，$w$）。
```
Comment (astonzhang) [inline chapter_linear-networks/image-classification-dataset.md:250]: 表示形状的括号使用英文半角()

File: chapter_linear-networks/image-classification-dataset.md [hunk 6]
```diff
@@ -241,19 +242,19 @@ for X, y in train_iter:
...
+* 数据迭代器是获得高性能的关键组件。依靠实现良好的数据迭代器，利用高性能计算来避免减慢训练循环。
 
 ## 练习
 
-1. 将 `batch_size`（实例，减少到 1）是否会影响读取性能？
+1. 将减少 `batch_size`（如减少到 1）是否会影响读取性能？
```
Comment (astonzhang) [inline chapter_linear-networks/image-classification-dataset.md:255]: 去掉“将”

File: chapter_linear-networks/image-classification-dataset.md [hunk 7]
```diff
@@ -241,19 +242,19 @@ for X, y in train_iter:
...
 1. 数据迭代器的性能非常重要。你认为当前的实现足够快吗？探索各种选择来改进它。
-1. 查看框架的在线 API 文档。还有哪些其他数据集可用？
+1. 查看框架的在线API文档。还有哪些其他数据集可用？
```
Comment (astonzhang) [inline chapter_linear-networks/image-classification-dataset.md:257]: 查阅

**Comment by mli** [comment]
Job d2l-zh/PR-556/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-556/

**Comment by mli** [comment]
Job d2l-zh/PR-556/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-556/

**Comment by mli** [comment]
Job d2l-zh/PR-556/6 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-556/

**Comment by mli** [comment]
Job d2l-zh/PR-556/7 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-556/

**Pull request merged**